RespokeSDK/NSString+urlencode.m

```diff
@@ -0,0 +1,39 @@
+//
+//  NSString+urlencode.m
+//  Respoke SDK
+//
+//  Copyright 2015, Digium, Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under The MIT License found in the
+//  LICENSE file in the root directory of this source tree.
+//
+//  For all details and documentation:  https://www.respoke.io
+//
+
+#import "NSString+urlencode.h"
+
+@implementation NSString (NSString_Extended)
+
+- (NSString *)urlencode {
```

Comment:
This isn't already in the iOS SDK? You should probably cite where you got the code in a comment here.